packages/theme/template/package.json

```diff
@@ -28,19 +28,19 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.0.0",
-    "@enact/i18n": "^4.0.0",
-    "@enact/spotlight": "^4.0.0",
-    "@enact/ui": "^4.0.0",
-    "classnames": "^2.2.6",
-    "prop-types": "^15.6.2",
-    "ramda": "^0.25.0",
-    "react": "^17.0.1"
+    "@enact/core": "^4.5.0-alpha.1",
+    "@enact/i18n": "^4.5.0-alpha.1",
+    "@enact/spotlight": "^4.5.0-alpha.1",
+    "@enact/ui": "^4.5.0-alpha.1",
+    "classnames": "^2.3.1",
+    "prop-types": "^15.8.1",
+    "ramda": "^0.28.0",
+    "react": "^18.0.0"
   },
   "peerDependencies": {
-    "ilib": "^14.6.0 || ^14.6.0-webos1"
+    "ilib": "^14.14.0 || ^14.14.0-webos1"
   },
   "devDependencies": {
-    "ilib": "^14.6.0"
+    "ilib": "^14.14.0"
```

Comment:
I think we need to add `eslint-config-enact-proxy` module here and update `eslintConfig` property to use it. Please refer to sandstone's package.json.

Reply:
i also updated travis.yml in order fot this change to work, but it's my first time when i work with travis configuration. Travis passes, but can you check if it can be done better? Thanks you!

Reply:
Let's update the travis.yml in #49.

Reply:
Oh, we had to do like you did :)